### ISSUE TITLE
Unknown browser query `dead` - WTF

### ISSUE THREAD

**Issue by xblue-light** [gatsbyjs/gatsby#8575]
I have a project using a Gatsby theme starter and last week I was building my project with no problems what so ever, this morning I tried to run a new build without having done any prior changes at all I get some intimidating error. I need to fix this this project is due to go live soon..

**Error Generating CSS failed
BrowserslistError: Unknown browser query `dead`**

FYI - Also getting this on some NEW starter Gatsby repos.

gatsby@2.4.1
node@10.11.0
npm@6.4.1

OS - Ubuntu 16.04.5 LTS

**Comment by xblue-light** [comment]
I managed to fix this ran in my terminal `$ node_modules/.bin/gatsby develop`

**Comment by kakadiadarpan** [comment]
@xblue-light can you provide relevant environment information by running `gatsby info --clipboard`?

**Comment by PierreBresson** [comment]
I also had the same issue... impossible to debug, working well on develop then crash on build. 

`gatsby info --clipboard` gives me that : 

 ```
 System:
    OS: macOS High Sierra 10.13.6
    CPU: x64 Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 8.12.0 - /usr/local/bin/node
    Yarn: 1.7.0 - /usr/local/bin/yarn
    npm: 6.4.1 - /usr/local/bin/npm
  Browsers:
    Chrome: 69.0.3497.100
    Firefox: 61.0.1
    Safari: 12.0
  npmPackages:
    gatsby: 1.9.266 => 1.9.266
    gatsby-link: 1.6.44 => 1.6.44
    gatsby-plugin-react-helmet: 2.0.11 => 2.0.11
    gatsby-plugin-react-next: 1.0.11 => 1.0.11
    gatsby-plugin-sass: 1.0.26 => 1.0.26
    gatsby-plugin-typography: 1.7.19 => 1.7.19
  npmGlobalPackages:
    gatsby-cli: 2.4.1
```

**Comment by m-allanson** [comment]
Thanks for the workaround @PierreBresson 👍 

This should be fixed in `gatsby-cli@2.4.2` and later.

**Comment by greglobinski** [comment]
I confirm installing `gatsby-cli@2.4.2` fixes problem for me
